<?xml version="1.0" encoding="utf-8"?>

<launch>

  <arg name="network_data_path"   value="/home/user/darknet" />

  <!-- Start darknet and ros wrapper -->
  <include file="$(find darknet_ros)/launch/darknet_ros.launch">
    <arg name="image"               value="/xtion/rgb/image_raw" />
    <arg name="yolo_weights_path"   value="$(arg network_data_path)/net/backup" />
    <arg name="yolo_config_path"    value="$(arg network_data_path)/net/cfg" />
    <arg name="ros_param_file"      value="$(find ek_challenger)/config/ros.yaml"/>
    <arg name="network_param_file"  value="$(find ek_challenger)/config/yolov4-airlab.yaml"/>
   </include>

</launch>
